spectator-ext-ipc/src/main/java/com/netflix/spectator/ipc/IpcTagKey.java

```diff
@@ -0,0 +1,158 @@
+/*
+ * Copyright 2014-2018 Netflix, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.netflix.spectator.ipc;
+
+/**
+ * Standard dimensions used for IPC metrics.
+ */
+public enum IpcTagKey {
+
+  /** The library that produced the metric. */
+  owner("owner"),
+
+  /**
+   * Indicates whether or not the request was successful. See {@link IpcResult}
+   * for permitted values.
+   */
+  result("ipc.result"),
+
+  /**
+   * Dimension indicating a high level cause for the request failure. These groups
+   * are the same for all implementations to make it easier to query across all
+   * services and client implementations. See {@link IpcErrorGroup} for permitted
+   * values.
+   */
+  errorGroup("ipc.error.group"),
+
+  /**
+   * Implementation specific error code.
+   */
+  errorReason("ipc.error.reason"),
```

Comment:
@kerumai you had some constants for various reasons. Are those needed since this field isn't really standardized?